ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -234,4 +242,19 @@ public void setEndpointUtil(EndpointUtil util) {
   public void setCqlQueryUtil(CqlQueriesImpl cqlQueryUtil) {
     this.cqlQueryUtil = cqlQueryUtil;
   }
+
+  private void addApplicableWarningsTo(CqlQueryResponse response) {
+    if (response.getQueryResponse() == null || response.getWarnings() == null) {
+      return;
```

Comment:
❓ Should we `debug` log here?

Reply:
I think so. Good catch.

Reply:
❓ Less is more, and data speaks louder than words. How about this instead?
```
LOGGER.debug("Could not add warnings to response, the response was null: {}", response);
```
and
```
LOGGER.debug("Could not add warnings to response, list of warnings was null: {}", response);
```

It minimizes the number of renames that have to occur if we refactor. Also be sure to test that calling `toString()` on any `CqlQueryResponse` object yields a useful object string, with all nested objects usefully written out as well. 

Typically when we log debug info we should also log relevant data for those troubleshooting the system.